src/sentry/services/hybrid_cloud/integration/__init__.py

```diff
@@ -299,7 +299,6 @@ def update_organization_integration(
 
     # The following methods replace instance methods of the ORM objects!
 
-    @rpc_method
     def get_installation(
```

Comment:
This one has to go over the wire, as Installations are only in Control Silo. Perhaps we can modify `provider.get_installation()` to take an id instead?

Reply:
When I put together the RPC methods for integrations, I believe the installations call the RPC, meaning calling get_installtion should be silo agnostic.

That means that `provider.get_installation().org_integration` [will make an RPC call](https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/base.py#L271-L278)